grid/app/main/sfl/cycles/worker_cycle.py

```diff
@@ -1,5 +1,5 @@
 import datetime
-from .. import db
+from ... import db
```

Comment:
The grouping "standard python, external module, local" is not being acknowledge here.